_test/common_functions/hashable_obj_tester.m

```diff
@@ -0,0 +1,56 @@
+function hashable_obj_tester(hobj,values,names)
+%hashable_obj_tester Generic function which validates correct hashable
+%object operations.
+%
+% Checks if all hashable properties invalidate hash, hash calculations work
+% and hash is stored-restored correctly
+%
+% Inputs:
+% hobj  -- instance of hashable object, better with proper values set
+% Optional
+% values-- cellarray of values to set to hashable object. If present, have
+%          to have number of elements equal to number of hashable
+%          properties and values of this properties compartible with
+%          serializeble, i.e. setting property do not contradict to other
+%          properties, validated through check_combo_arg method
+% names -- list of the properties to set to check hashable object.
+%          If missing, use hashableFields
+%
+if ~exist('names','var')
+    flds = hobj.hashableFields();
+else
+    flds = names;
+end
+
+
+n_flds = numel(flds);
+if ~exist('values','var')
+    values = cellfun(@(x)hobj.(x),flds,'UniformOutput',false);
+end
+
+if ~hobj.hash_defined
+    hobj = hobj.build_hash();
+end
+
+for i=1:n_flds
+    assertTrue(hobj.hash_defined)
+    hobj.(flds{i}) = values{i};
+    if hobj.hash_defined
+        assertFalse(true, ...
+            sprintf('*** Hash remains defined for changed property: %s ',flds{i}));
+    end
```

Comment:
unclear what this is doing. assertFalse(true...) looks strange

Reply:
`assertFalse(hobj.hash_defined,sprinft())` would do but this form will print each time `assertFalse` is called in the loop regardless of failure. 
I want to save prining in a loop (old fashoned, I am still remember that it considered very inefficient) so validating `hash_defined` separately. If it fails, error is reported through usual `assert` channel redirected appropriately on every test system.